docs/source-pytorch/visualize/supported_exp_managers.rst

```diff
@@ -68,13 +68,13 @@ To use `Neptune.ai <https://neptune.ai/>`_ first install the neptune package:
 
 .. code-block:: bash
 
-    pip install neptune-client
+    pip install neptune
 
 or with conda:
 
 .. code-block:: bash
 
-    conda install -c conda-forge neptune-client
+    conda install -c conda-forge neptune
```

Comment:
The conda-forge package has not been renamed yet (I believe this is in progress), so this currently does not work.

Reply:
Sorry. Can you revert those changes then?